lib/v2/search.js

```diff
@@ -91,6 +96,16 @@ const Search = class Search {
     return this;
   }
 
+  fields(value) {
```

Comment:
It's possible to get a unique set of values from an array by using `Array.from(new Set(this.query_hash_fields))`.

I know it's the same in more places here so just a hint that maybe it would reduce the boilerplate of having each field doing this `find`ing and `push`ing.

Reply:
Thanks for the suggestion! Changed the implementation accordingly.